### PR TITLE
Add unique title to individual experience pages

### LINKED ISSUE #4440: Remove Title from individual forms
The title "My experience and skills" and it's line of copy underneath create confusion when they repeat inside individual experience forms.

## To do:
Remove the title and text from the forms and keep it only on the main experience section

chrome_ercqgM6yUp

## Acceptance Criteria
- [ ] ...

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap

### PR DESCRIPTION
**Pull request by petertgiles**
~~Removes the title from the individual experience pages.~~
Adds a unique title to individual experience pages.

Closes #4440

### CODE REVIEW

**Review by esizer — CHANGES_REQUESTED** [review]
Hmm, I'm not sure I agree with this. If users are confused by the title, I think we should change it and not fully remove it. 

Each page should have one unique `<h1>` that acts as the document title describing the content on the page. This also introduces the issue of heading ranks being nested improperly due to `<h2>` now being the first heading on the page.

Could we use something like "Edit/Add Experience" as the title rather than removing it all together?

REF: https://www.a11yproject.com/posts/how-to-accessible-heading-structure/

**Comment by mnigh** [comment]
> Each page should have one unique `<h1>` that acts as the document title describing the content on the page. This also introduces the issue of heading ranks being nested improperly due to `<h2>` now being the first heading on the page.

Agree with @esizer, see axe Dev Tools for reinforcement:

### Screenshot
Screen Shot 2022-11-01 at 10 12 40

**Comment by petertgiles** [comment]
Good call.  I'll bounce it back to Jerbo.

**Comment by Jerryescandon** [comment]
image

**Comment by petertgiles** [comment]
OK, they're in.  I noticed that two of the 
[translations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4537/commits/2afaa456b7e39aa05e18ff36cb72e855acef06f6) we received are a bit inconsistent.  @gobyrne could you take a quick look at these?
image
image

File: frontend/talentsearch/src/js/components/experienceForm/ExperienceForm.tsx [hunk 1]
```diff
@@ -147,19 +147,46 @@ export const ExperienceForm: React.FunctionComponent<ExperienceFormProps> = ({
...
+        return intl.formatMessage({
+          defaultMessage: "Edit work experience",
+          id: "Ytlyzb",
+          description: "Page title for the work experience profile form",
+        });
+      default:
+        return "";
+    }
+  };
+
```
Comment (esizer) [inline frontend/talentsearch/src/js/components/experienceForm/ExperienceForm.tsx:186]: Do we also want a case for if the user is adding a new one or editing an existing one? 😅

**Comment by esizer** [comment]
Marked requested changes as completed. I think we can merge once the conflict is resolbed based on the convo during standup.

**Comment by gobyrne** [comment]
Eeak, sorry I'm so late. Can we use "Modifier l'expérience des études" and "Modifier l'expérience communautaire" please?

**Pull request merged**